### DISCUSSION TITLE
Commit Message Formatting on github.com

### DISCUSSION THREAD

**Discussion by openbl** [community/community#12114] [Other Feature Feedback, Questions, & Ideas]
I'd love to be able to see the 72 character limit in commit message textareas when writing commit messages. 

I know folks have different expectations of commit message format but my team does try to conform to that. So when I update a message when I'm merging in a PR on github.com I'm unable to (or unaware of how) to do that w/o leaving the website.

Thanks!

**Comment by benipeled** [comment]
+1,
IMO, it should be optional with an option to set the <int> characters
